recipes/mvlearn/LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
I do not believe you need to manually package the license file since it's included in the tarball from github

Reply:
Removed the LICENSE file, thanks!